worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -20,15 +20,63 @@ if(!ACTIONS){
 }
 
 var URL = Java.type("java.net.URL");
+var MDC = Java.type("org.slf4j.MDC");
+var UUID = Java.type("java.util.UUID");
 
-function onProcessTask() {
+function onProcessTask(e) {
+    addMdcLoggingData(e);
     thisScript.install();
 }
 
+function addMdcLoggingData(e) {
+    // The logging pattern we use uses a tenantId and a correlationId:
+    // 
+    // https://github.com/CAFapi/caf-logging/tree/4ef35ae3a6da4329a427667782f7aaff4fee8c1d#pattern
+    // https://github.com/CAFapi/caf-logging/blob/4ef35ae3a6da4329a427667782f7aaff4fee8c1d/src/main/resources/logback.xml#L27
```

Comment:
You can refer to the release tags now rather than the commit ids:

https://github.com/CAFapi/caf-logging/tree/v1.0.0#pattern
etc.